frontend/vite-prod.config.ts

```diff
@@ -40,17 +43,17 @@ export default defineConfig(({ command }) => {
       emptyOutDir: true,
       sourcemap: true,
       minify: false,
+      // cssMinify: 'lightningcss',
```

Comment:
Remove?

Reply:
Fixed in a91d35396e67b20d8a641f240b9de9742187151b